trin-core/src/utils.rs

```diff
@@ -40,6 +41,13 @@ pub fn get_default_data_dir(local_enr: Enr) -> String {
     }
 }
 
+pub fn setup_overlay_db(local_enr: Enr) -> DB {
+    let data_path = get_data_dir(local_enr);
+    let mut db_opts = Options::default();
+    db_opts.create_if_missing(true);
+    DB::open(&db_opts, data_path).unwrap()
```

Comment:
I'm trying to resist to error handle the database stuff for now, because there is a PR in progress for the DB layer and we will have to refactor this anyway really soon..